### PR TITLE
elastic_search_visitor: nested author queries

### PR DESCRIPTION
**Pull request by iulianav**
Make author queries nested in order to be compatible with the 'authors' field which has been changed to nested.

Signed-off-by: Iuliana Voinea <iulianavoinea96@gmail.com>

### CODE REVIEW

File: inspire_query_parser/visitors/elastic_search_visitor.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ class ElasticSearchVisitor(Visitor):
     AUTHORS_NAME_VARIATIONS_FIELD = 'authors.name_variations'
     AUTHORS_BAI_FIELD = 'authors.ids.value'
     BAI_REGEX = re.compile(r'^((\w|-|\')+\.)+\d+$', re.UNICODE | re.IGNORECASE)
-    JOURNAL_NESTED_QUERY_PATH = 'publication_info'
```
Comment (drjova) [inline inspire_query_parser/visitors/elastic_search_visitor.py]: why? :)

Comment (iulianav) [reply inspire_query_parser/visitors/elastic_search_visitor.py]: @drjova it is not being used anywhere.

Comment (michamos) [reply inspire_query_parser/visitors/elastic_search_visitor.py]: but shouldn't it? https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/records/mappings/records/hep.json#L724

Comment (iulianav) [reply inspire_query_parser/visitors/elastic_search_visitor.py]: We are actually using this one https://github.com/inspirehep/inspire-query-parser/pull/90/files/ee635c8566688e7f61493ca3fc548cdc5f074589#diff-113310c03456148bafc0ddbf0b89dde5R76

**Pull request merged**